### PR TITLE
feat(@wallet): sort saved addresses

### LINKED ISSUE #9772: Wallet -> Saved addresses: adding a new saved address should move it to the top of the list
### Description

Adding a new saved address should move it to the top of the list

image

### PR DESCRIPTION
**Pull request by alaibe**
fixes #9772

status-go pr: https://github.com/status-im/status-go/pull/3265

### CODE REVIEW

File: src/app_service/service/saved_address/service.nim [hunk 1]
```diff
@@ -50,6 +53,7 @@ proc fetchAddresses(self: Service) =
         except:
           continue
 
+    sort(self.savedAddresses, savedAddressSort, order = SortOrder.Descending)
```
[the remark this answers is not in the document]
Comment (alaibe) [reply src/app_service/service/saved_address/service.nim]: updated

File: ui/app/AppLayouts/Wallet/views/SavedAddressesView.qml [hunk 2]
```diff
@@ -107,7 +108,10 @@ Item {
         anchors.left: parent.left
         visible: listView.count > 0
         spacing: 5
-        model: RootStore.savedAddresses
+        model: SortFilterProxyModel {
+            sourceModel: RootStore.savedAddresses
+            sorters: RoleSorter { roleName: "createdAt"; sortOrder: Qt.DescendingOrder }
```
Comment (caybro) [inline ui/app/AppLayouts/Wallet/views/SavedAddressesView.qml:113]: Yay, much better than sorting in the backend :100:

**Review by caybro — APPROVED** [review]

File: ui/app/AppLayouts/Wallet/views/SavedAddressesView.qml (hunk 2 above)
Comment (saledjenic) [reply ui/app/AppLayouts/Wallet/views/SavedAddressesView.qml:113]: Isn't this at the end of the day a backend sort? 😉

Comment (caybro) [reply ui/app/AppLayouts/Wallet/views/SavedAddressesView.qml:113]: Well yes :) But this is parametrized sort that can be different in different QML files, if needed ;)

**Comment by anastasiyaig** [comment]
@IvanBelyakoff what i did - i just installed this pr and added 3 addresses there today. Funny thing that when opening the DB file and trying to access `saved addresses` table the SQLLite crashes

**Comment by caybro** [comment]
Needs a rebase

**Pull request merged**